packages/components/src/Pagination/_docs/Pagination.mdx

```diff
@@ -0,0 +1,28 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as PaginationStories from "./Pagination.stories"
+
+<Meta of={PaginationStories} />
+
+# Pagination
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Pagination"
+  figma="https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%EF%B8%8F%F0%9F%96%BC%EF%B8%8F-Component-Gallery?node-id=9%3A39913&t=P1w10jr2cpPuaayw-1"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3082092975/Pagination"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { Pagination } from "@kaizen/components"'
+/>
+
+## Overview
+
+Pagination separates large bodies of content into separate pages. You can access each page via a shared index of links.
```

Comment:
Nice 👍 
Does this one need an API section as well though?

Reply:
We've switch to low-effort mode, if it didn't have any old docs, then we're skipping it in the migration.